include/openthread/platform/border_routing.h

```diff
@@ -65,6 +65,23 @@ extern "C" {
  */
 extern void otPlatBorderRoutingProcessIcmp6Ra(otInstance *aInstance, const uint8_t *aMessage, uint16_t aLength);
 
+/**
+ * Process a prefix received from the DHCPv6 PD Server. The prefix is received on
+ * the DHCPv6 PD client callback and provided to the Routing Manager via this
+ * API.
+ *
+ * The lifetime of the prefix can be updated by calling the function again with updated
+ * time values.
+ *
+ * * Requires `OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE`.
```

Comment:
```suggestion
 * Requires `OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE`.
```

For consistency with the other API above, and because we don't need a bullet for one item.